doc/data/messages/a/arguments-differ/good.py

```diff
@@ -0,0 +1,23 @@
+class Rectangle:
+    def __init__(self, length, width):
+        self.length = length
+        self.width = width
+
+
+class Square(Rectangle):
+    def __init__(self, length, width):
+        self.length = length
+        self.width = width
```

Comment:
This "feel" wrong, maybe 
```suggestion
class Square(Rectangle):
    def __init__(self, length, width=None):
        if width == None:
            width = length
        self.length = length
        self.width = width
```

Reply:
Ha, you're right I missed that. I feels the design is wrong because this is not how a square should be enforced.

```suggestion
class Square(Rectangle):
    def __init__(self, length, width):
        self.length = length
        self.width = width
        assert length == width
```
Maybe ?

Reply:
Let's do a circle then? 😄 

I'm fine with the `assert` but it kind of takes the attention away from the "goal" of the example. Like, if we need redundant code to make the example work perhaps we should change the example.

Reply:
Hmm actually, for PlasticDuck the fix I wanted to convay was to create two classes that are not inheriting from each other. There's not much sense in shoe horning an abstraction like this imo.